Refactoring

TeaDove opened the pull request:
- add Makefile for crosscompile
- bump go version to go1.20
- minor code logic refactoring

Commit:
ref: minor code refactoring

Commit:
ref: fix switch

Commit:
ref: fix switch

Commit:
ref: add makefile for crosscompile

esimov commented:
Looks good to me, but I would like the package name to be preserved. Currently the package name is defined as `github.com/teadove/gobrot`, however I would like to remain as `github.com/esimov/gobrot`.

Commit:
ref: rename package

TeaDove commented:
> Looks good to me, but I would like the package name to be preserved. Currently the package name is defined as `github.com/teadove/gobrot`, however I would like to remain as `github.com/esimov/gobrot`.

Oh, yes, of course, I have changed package for my account to test it locally

Changed back to `github.com/esimov/gobrot`
